package.json

```diff
@@ -27,24 +27,32 @@
     "email": "peter.han.9493@gmail.com",
     "url": "https://github.com/peterthehan"
   },
-  "main": "index.js",
-  "bin": "index.js",
+  "main": "src/index.ts",
+  "bin": "ts-node src/index.ts",
   "scripts": {
     "bot": "node app/src/index.js",
-    "check": "prettier --no-config --check .",
-    "dry": "node index.js --dry-run",
+    "build": "npx tsc",
+    "check": "npx eslint . && prettier --no-config --check .",
```

Comment:
`check` and `lint` are meant to be used in the context of github actions (you'll see that it's called from there) and since they have their own outputs we shouldn't combine them like this.

check should be reverted back to:
```
"check": "prettier --no-config --check .",
```